app/controllers/providers/check_provider_answers_controller.rb

```diff
@@ -25,6 +25,10 @@ def continue
           legal_aid_application.lead_linked_application.destroy!
           legal_aid_application.update!(copy_case: nil, copy_case_id: nil)
         end
+        if applicant.under_18?
```

Comment:
i can see @applicant being set (for the index action only) but cannot see where the applicant method is defined. Is there an attr_writer for `applicant` being inherited somewhere or somesuch. Line 28 is using the fully resolved applicant `legal_aid_application.applicant`.

Anyway this would seem to be all that is required but be good to run it past other devs.